__tests__/app.test.js

```diff
@@ -86,7 +88,12 @@ describe('/api/articles', () => {
             .get('/api/articles')
             .expect(200)
             .then(({ body }) => {
-                body.articles.forEach(article => {
+                const articles = body.articles;
+                const dates = articles.map(article => new Date(article.created_at).getTime());
+
+                expect(articles.length).toBeGreaterThan(0);
```

Comment:
Better to test for the exact length to see that you are getting all the data you expect, you can count the length in the data file